pysam/libcalignedsegment.pyx

```diff
@@ -2448,8 +2442,9 @@ cdef class PileupColumn:
             # warning: there could be problems if self.n and self.buf are
             # out of sync.
             for x from 0 <= x < self.n_pu:
-                pileups.append(makePileupRead(&(self.plp[0][x]),
-                                              self._alignment_file))
+                pileups.append(
```

Comment:
... just habit, 120 is ok. Which reminds me that I want to turn on pep8 tests at one point.

I am short-sighted and usually work with a extra large font-size and an emacs split screen having two windows side-by-side (such as code | tests). Need better glasses :=).